sdk/boards/sonata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "devices": {
+        "clint": {
+            "start": 0x2000000,
+            "length":  0x10000
+        },
+        "uart": {
+            "start": 0x81000000,
+            "length":    0x1000
```

Comment:
This doesn't seem to match the size of the UART device that you've added (should be 52 bytes?)

Reply:
Yep, you're spot on. I'll correct that